src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -147,6 +147,7 @@ var HTMLDOMPropertyConfig = {
     srcDoc: 0,
     srcLang: 0,
     srcSet: 0,
+    srcObject: 0,
```

Comment:
I'm not sure I follow here. What is the difference between property and an attribute?

`srcObject` is no different to `src` really, the only issue is that one accepts a stream while the other a url.